Update docs for SP2021 game (#174)
sourcebots/docs: content/api/magnet-board.md

---

> title: Magnet Board API

Reviewer: I don't think it really makes sense to call this a "board".

Author: Done :)

weight: 8
---
 
The magnet board is only available on the [Crane robot](../../robots/crane).
 

> The magnet of the crane will stick to any magnetic objects (especially containers).

Reviewer: ```suggestion
The magnet of the crane will only stick to containers.
```

 
## Getting the magnet's status
 
A magnet can be either energised or de-energised, you can turn the magnet on or off by setting `r.magnet.energised` like so:
```python
# Turn on the magnet
r.magnet.energised = True
# Turn off the magnet
r.magnet.energised = False
```

{{% notice warning %}}
Check the spelling of `energised`, it's in british english, `energized` will not work!
{{% /notice %}}

You can get the current state of the magnet by getting the value of `energized` like so:

```python
magnet_on = r.magnet.energised

> ```

Reviewer: There's also the very useful `nearby` API which is worth documenting https://github.com/sourcebots/competition-simulator/blob/sb2021/modules/sr/robot/magnet.py#L25
